gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -108,6 +108,16 @@ const NxToastPage = () =>
                 be set up internally by <NxCode>NxToast</NxCode>. Calling code should instead use the
                 {' '}<NxCode>onClose</NxCode> prop of <NxCode>NxToast</NxCode> itself. Additionally, note that the alert
                 text content should be brief: only one rendered line of text per toast is supported.
+                <NxWarningAlert>
+                  <NxCode>Nx*Alert</NxCode>s must be passed a {' '}
+                  <NxTextLink external href="https://www.w3.org/TR/wai-aria/#live_region_roles">
+                    Live Region
+                  </NxTextLink>
+                  {' '}role if not already provided (see
+                  {' '}<NxCode><NxTextLink href="#/pages/Alert">NxAlert</NxTextLink></NxCode>
+                  for more information about default roles). Without a designated role, the content inside
+                  {' '}<NxCode>NxToast</NxCode> will not be immediately read by screenreaders when rendered on the page.
+                </NxWarningAlert>
```

Comment:
I'm wondering if alerts in `toasts` should just always use the `alert` role.  Regardless of the level of toast used, that role seems like the best fit for a component that pops up as the result of some event like this.  We could even have `NxToast` pass down the `alert` role automatically unless another role is specified by the caller

Reply:
> I'm wondering if alerts in `toasts` should just always use the `alert` role. Regardless of the level of toast used, that role seems like the best fit for a component that pops up as the result of some event like this. We could even have `NxToast` pass down the `alert` role automatically unless another role is specified by the caller

That makes sense to me. With everything set to alert, aria-live will always be set to assertive and the screenreader will interrupt itself to read the toast message. As long as you're good with that, then I'll get started on changing it. So NxToast would pass it down to the alert in the NxToast.tsx file, correct?

Reply:
> So NxToast would pass it down to the alert in the NxToast.tsx file, correct?

Yes, but it should probably done it in such a way that the if the caller sets a role on the `NxAlert` we keep that role instead

Reply:
> > So NxToast would pass it down to the alert in the NxToast.tsx file, correct?
> 
> Yes, but it should probably done it in such a way that the if the caller sets a role on the `NxAlert` we keep that role instead

Implemented here: [3d8d907](https://github.com/sonatype/sonatype-react-shared-components/pull/766/commits/3d8d907b09f057b1ac1feb0e3f23346e32a19ace). Since updating the role of NxAlert is not required, I removed AlertComponentProps in the 'Simple' and 'Legacy' examples.